core/templates/dev/head/components/RatingComputationService.js

```diff
@@ -46,6 +46,14 @@ oppia.factory('RatingComputationService', [function() {
 
         return totalValue / totalNumber;
       }
+    },
+    getRelativeChangeInAverageRating: function(newRating, oldRating) {
+      if (!oldRating) {
+        return '+' + parseFloat(newRating).toFixed(2);
```

Comment:
Ah, what I mean is that since numbers already have the `toFixed` method (see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed)) you should just be able to call `newRating.toFixed(2)`